tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -8,6 +8,9 @@ Param(
 $ErrorActionPreference = 'Stop';
 Set-Location c:\mnt
 
+# Install chocolatey binary
+$env:chocolateyUseWindowsCompression = 'true'; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
I thought Windows core doesn't have the COM for the compression. Are you sure this script work on Windows Core container?

Reply:
I've checked the pipeline that kylian triggered with this change and it does seem to work. Maybe you are talking about Windows Nano and not Windows Core?